### PR TITLE
Add some logic to detect the case when a signature already exists.

### PR DESCRIPTION
**Pull request by dlorenc**
cc @priyawadhwa

### CODE REVIEW

**Review by priyawadhwa — APPROVED** [review]
nice!

File: pkg/cosign/tlog/verify.go [hunk 1]
```diff
@@ -36,13 +38,18 @@ import (
 
 // Verify will verify the signature, public key and payload are in the tlog, as well as verifying the signature itself
 // most of this code taken from github.com/sigstore/rekor/cmd/cli/app/verify.go
-func Verify(signedPayload []cosign.SignedPayload, publicKey string) error {
-	if os.Getenv(Env) != "1" {
+func Verify(signedPayload []cosign.SignedPayload, publicKey ed25519.PublicKey) error {
+	der, err := x509.MarshalPKIXPublicKey(publicKey)
+	if err != nil {
 		return nil
```
Comment (priyawadhwa) [inline pkg/cosign/tlog/verify.go:44]: `return err`?

Comment (dlorenc) [reply pkg/cosign/tlog/verify.go]: :)

**Review by priyawadhwa — APPROVED** [review]
LGTM

**Pull request merged**